### ISSUE TITLE
sigmatools 0.12 broken on pypi

### ISSUE THREAD

**Issue by EccoTheFlintstone** [Neo23x0/sigma#409]
sigmatools wheel 0.12 on pypi is missing "**modifiers**", and **sigmac** cannot start

(probably missing **'sigma.parser.modifiers'** from here : https://github.com/Neo23x0/sigma/blob/master/tools/setup.py#L37)
```
$ sudo python3 -m pip install sigmatools

Collecting sigmatools
  Downloading https://files.pythonhosted.org/packages/2f/18/7785b6f6f57dde2af2a756c696aa3623240ccf462161bd87d0029d493957/sigmatools-0.12-py3-none-any.whl (92kB)
    100% |████████████████████████████████| 102kB 2.8MB/s 
...
Successfully installed deprecated-1.2.6 jsonschema-3.0.2 pymisp-2.4.112 pyrsistent-0.15.4 sigmatools-0.12
```

```
$ sigmac
Traceback (most recent call last):
  File "/usr/local/bin/sigmac", line 25, in <module>
    from sigma.parser.collection import SigmaCollectionParser
  File "/usr/local/lib/python3.6/dist-packages/sigma/parser/collection.py", line 18, in <module>
    from .rule import SigmaParser
  File "/usr/local/lib/python3.6/dist-packages/sigma/parser/rule.py", line 20, in <module>
    from .modifiers import apply_modifiers

ModuleNotFoundError: No module named 'sigma.parser.modifiers'
```

There's indeed no **modifiers** directory in installed data:
```
ls -al  /usr/local/lib/python3.6/dist-packages/sigma/parser/
total 60
drwxr-sr-x 3 root staff  4096 août   5 13:53 ./
drwxr-sr-x 6 root staff  4096 août   5 13:53 ../
-rw-r--r-- 1 root staff  3304 août   5 13:53 base.py
-rw-r--r-- 1 root staff  3471 août   5 13:53 collection.py
-rw-r--r-- 1 root staff 26416 août   5 13:53 condition.py
-rw-r--r-- 1 root staff   817 août   5 13:53 exceptions.py
drwxr-sr-x 2 root staff  4096 août   5 13:53 __pycache__/
-rw-r--r-- 1 root staff  7516 août   5 13:53 rule.py
```

And also missing from wheel published on pypi
```
7z l /tmp/sigmatools-0.12-py3-none-any.whl

7-Zip [64] 16.02 : Copyright (c) 1999-2016 Igor Pavlov : 2016-05-21
p7zip Version 16.02 (locale=fr_FR.UTF-8,Utf16=on,HugeFiles=on,64 bits,12 CPUs Intel(R) Core(TM) i9-8950HK CPU @ 2.90GHz (906EA),ASM,AES-NI)

Scanning the drive for archives:
1 file, 92320 bytes (91 KiB)

Listing archive: /tmp/sigmatools-0.12-py3-none-any.whl

--
Path = /tmp/sigmatools-0.12-py3-none-any.whl
Type = zip
Physical Size = 92320

   Date      Time    Attr         Size   Compressed  Name
------------------- ----- ------------ ------------  ------------------------
2018-01-26 22:14:08 .....            0            2  sigma/__init__.py
2019-05-19 22:27:58 .....        13108         3198  sigma/configuration.py
2019-01-14 21:19:54 .....         4768         1345  sigma/filter.py
2019-07-31 21:44:10 .....         1403          746  sigma/tools.py
2019-05-16 21:09:26 .....         7125         2384  sigma/backends/ala.py
2019-04-25 21:42:12 .....         6771         2024  sigma/backends/arcsight.py
2019-07-31 21:44:10 .....        13030         3418  sigma/backends/base.py
2019-03-15 23:49:52 .....        19178         1930  sigma/backends/data.py
2019-07-31 21:44:10 .....         1257          661  sigma/backends/discovery.py
2019-07-31 21:44:10 .....        43984         9750  sigma/backends/elasticsearch.py
2018-07-26 22:02:58 .....         1164          609  sigma/backends/exceptions.py
2019-04-22 21:22:48 .....         1416          776  sigma/backends/graylog.py
2019-02-25 22:16:18 .....         2036          976  sigma/backends/logpoint.py
2019-04-25 21:42:12 .....         2358         1006  sigma/backends/misc.py
2019-05-20 22:32:58 .....         2895         1237  sigma/backends/mixins.py
2019-04-25 21:42:12 .....         4516         1490  sigma/backends/netwitness.py
2019-04-25 21:42:12 .....         8458         2498  sigma/backends/powershell.py
2019-04-25 21:42:12 .....         8690         2256  sigma/backends/qradar.py
2018-08-02 20:44:24 .....         4138         1385  sigma/backends/qualys.py
2019-02-25 22:16:18 .....         7573         2223  sigma/backends/splunk.py
2019-07-31 21:44:02 .....        13685         3971  sigma/backends/sumologic.py
2019-04-25 21:42:12 .....         2846         1099  sigma/backends/tools.py
2019-05-16 21:09:26 .....        10546         3099  sigma/backends/wdatp.py
2019-05-24 20:32:30 .....         3769         1482  sigma/config/collection.py
2019-03-01 23:50:16 .....          885          489  sigma/config/exceptions.py
2019-04-25 21:42:12 .....        10418         2619  sigma/config/mapping.py
2019-03-01 23:50:16 .....         3304         1405  sigma/parser/base.py
2018-08-14 19:36:02 .....         3471         1310  sigma/parser/collection.py
2019-07-31 21:44:02 .....        26416         6531  sigma/parser/condition.py
2018-07-26 22:02:58 .....          817          466  sigma/parser/exceptions.py
2019-07-31 21:44:10 .....         7516         2244  sigma/parser/rule.py
2019-05-19 22:44:32 .....         2276          475  sigmatools-0.12.data/data/etc/sigma/arcsight.yml
2019-07-31 21:49:00 .....          178          116  sigmatools-0.12.data/data/etc/sigma/filebeat-defaultindex.yml
2019-07-31 21:44:02 .....         4758         1430  sigmatools-0.12.data/data/etc/sigma/helk.yml
2019-07-31 21:49:00 .....         3395          637  sigmatools-0.12.data/data/etc/sigma/logpoint-windows.yml
2019-07-31 21:49:00 .....          172          119  sigmatools-0.12.data/data/etc/sigma/logstash-defaultindex.yml
2019-07-31 21:49:00 .....          528          262  sigmatools-0.12.data/data/etc/sigma/logstash-linux.yml
2019-07-31 21:49:00 .....          985          319  sigmatools-0.12.data/data/etc/sigma/logstash-windows.yml
2019-05-19 22:44:08 .....         1694          485  sigmatools-0.12.data/data/etc/sigma/netwitness.yml
2019-07-31 21:49:00 .....         1810          399  sigmatools-0.12.data/data/etc/sigma/powershell.yml
2019-05-19 22:48:04 .....          704          295  sigmatools-0.12.data/data/etc/sigma/qradar.yml
2019-05-19 22:45:10 .....          440          189  sigmatools-0.12.data/data/etc/sigma/qualys.yml
2019-07-31 21:49:00 .....          200          128  sigmatools-0.12.data/data/etc/sigma/splunk-windows-index.yml
2019-07-31 21:49:00 .....         1909          424  sigmatools-0.12.data/data/etc/sigma/splunk-windows.yml
2019-05-19 22:42:24 .....         1422          455  sigmatools-0.12.data/data/etc/sigma/sumologic.yml
2019-05-19 22:46:40 .....         2270          615  sigmatools-0.12.data/data/etc/sigma/thor.yml
2019-07-31 21:49:00 .....         4041         1148  sigmatools-0.12.data/data/etc/sigma/winlogbeat-old.yml
2019-07-31 21:49:00 .....         4554         1173  sigmatools-0.12.data/data/etc/sigma/winlogbeat.yml
2019-05-19 22:27:58 .....          268          145  sigmatools-0.12.data/data/etc/sigma/generic/sysmon.yml
2019-05-19 22:27:58 .....          409          209  sigmatools-0.12.data/data/etc/sigma/generic/windows-audit.yml
2019-08-01 21:46:04 .....         1313          729  sigmatools-0.12.data/scripts/merge_sigma
2019-08-01 21:46:04 .....         2286          972  sigmatools-0.12.data/scripts/sigma2misp
2019-08-01 21:46:04 .....        12311         3779  sigmatools-0.12.data/scripts/sigmac
2019-08-01 21:47:16 .....          465          283  sigmatools-0.12.dist-info/DESCRIPTION.rst
2019-08-01 21:47:16 .....         1179          640  sigmatools-0.12.dist-info/metadata.json
2019-08-01 21:47:16 .....            6            8  sigmatools-0.12.dist-info/top_level.txt
2019-08-01 21:47:16 .....           92           92  sigmatools-0.12.dist-info/WHEEL
2019-08-01 21:47:16 .....         1527          763  sigmatools-0.12.dist-info/METADATA
2019-08-01 21:47:16 .....         5420         2770  sigmatools-0.12.dist-info/RECORD
------------------- ----- ------------ ------------  ------------------------
2019-08-01 21:47:16             294153        83688  59 files

```

**Comment by thomaspatzke** [comment]
Indeed, it's broken. Sorry for that, I'm fixing this now!

**Comment by thomaspatzke** [comment]
Fixed in 0.12.1, thanks for the notification!
